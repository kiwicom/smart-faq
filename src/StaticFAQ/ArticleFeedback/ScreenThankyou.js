// @flow

import * as React from 'react';
import css from 'styled-jsx/css';
import { Heading } from '@kiwicom/orbit-components';
import { Close } from '@kiwicom/orbit-components/lib/icons';

import screensList from './screensList';
import { Box } from '../../common';

type Props = {|
  changeScreen: (nextScreen: string) => void,
|};

const style = css`
  div.feedbackMessage {
    text-align: center;
    padding: 32px 83px;
  }
  div.close-icon {
    position: absolute;
    top: 8px;
    right: 8px;
    cursor: pointer;
  }
`;

const ScreenThankyou = (props: Props) => {
  return (
    <Box border="none" borderRadius="4px" backgroundColor="#f5f7f9">
      <div className="feedbackMessage">
        <div
          className="close-icon"
          onClick={() => props.changeScreen(screensList.INITIAL)}
          onKeyUp={null}
          tabIndex="-1"
          role="button"
        >
          <Close customColor="#bac7d5" size="small" />
        </div>
        <Heading type="title2">Thanks, we appreciate your insight.</Heading>
        <style jsx>{style}</style>
      </div>
    </Box>
  );
};

export default ScreenThankyou;
