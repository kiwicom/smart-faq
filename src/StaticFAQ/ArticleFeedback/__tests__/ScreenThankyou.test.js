// @flow

import * as React from 'react';
import { shallow } from 'enzyme';

import ScreenThankyou from '../ScreenThankyou';

describe('SignIn', () => {
  const component = shallow(<ScreenThankyou />);
  it('should match snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
