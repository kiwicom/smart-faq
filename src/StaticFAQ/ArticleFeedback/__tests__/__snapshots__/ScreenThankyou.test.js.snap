// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScreenThankyou should match snapshot 1`] = `
<Box
  backgroundColor="#f5f7f9"
  border="none"
  borderRadius="4px"
>
  <div
    className="jsx-3762548811 thank-you"
  >
    <div
      className="jsx-3762548811 close-icon"
      onClick={[Function]}
      onKeyUp={null}
      role="button"
      tabIndex="-1"
    >
      <Close
        customColor="#bac7d5"
        size="small"
      />
    </div>
    <WithTheme(Heading)
      type="title3"
    >
      Thanks again.
    </WithTheme(Heading)>
    <div
      className="jsx-3762548811 message-2"
    >
      <WithTheme(Heading)
        type="title3"
      >
        We appreciate your insight.
      </WithTheme(Heading)>
    </div>
    <JSXStyle
      css="div.thank-you.jsx-3762548811{display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-flex-direction:column;-ms-flex-direction:column;flex-direction:column;-webkit-box-pack:center;-webkit-justify-content:center;-ms-flex-pack:center;justify-content:center;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;height:251px;}div.close-icon.jsx-3762548811{position:absolute;top:8px;right:8px;cursor:pointer;}div.message-2.jsx-3762548811{margin-bottom:16.6px;}"
      styleId="3762548811"
    />
  </div>
</Box>
`;
